internal/testutil/reindent.go

```diff
@@ -38,3 +38,16 @@ func Reindent(in string) []byte {
 	}
 	return buf.Bytes()
 }
+
+// IndentLines indents each line in the provided string with the prefix.
```

Comment:
[Nit]: What about `PrefixEachLine`, the function can be used for more things than indenting, right?